tests/test_simple_hierarchy.py

```diff
@@ -18,29 +18,29 @@ class Delta(Charlie):
 
 
 def test_tracked_descendants():
```

Comment:
Isn't this a duplicate test ? (Also present above in, test_explicit_keys.py)

Reply:
Not duplicate, no - the setup is different, as are the assertions. They're testing the different results of the same method under different circumstances.